src/porepy/models/constitutive_laws.py

```diff
@@ -230,7 +230,7 @@ def aperture(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
                 f_max = pp.ad.Function(pp.ad.maximum, "maximum_function")
 
                 a_ref = self.residual_aperture(subdomains_of_dim)
-                apertures_of_dim = f_max(normal_jump, a_ref)
+                apertures_of_dim = f_max(normal_jump + a_ref, a_ref)
```

Comment:
Can we document that the normal jump will be added to the reference aperture in the function documentation?